recipe/meta.yaml

```diff
@@ -96,6 +96,7 @@ requirements:
 test:
   imports:
     - wx
+    - wx.lib.pubsub
```

Comment:
:+1: 

Feel free to add yourself as a maintainer on your next commit ;-p

Reply:
Done :)